vkubelet/pod.go

```diff
@@ -12,11 +13,14 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/client-go/util/workqueue"
 )
 
+const softDeleteKey = "com.github.virtualKubelet.virtualKubelet.softDelete"
```

Comment:
As an annotation key, this format seems weird to me. Given there's the `virtual-kubelet.io` domain too, I suggest changing to `softdelete.pod.virtual-kubelet.io`.

Reply:
What about:
virtual-kubelet.io/softDelete?

That jives with: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/

> Annotations are key/value pairs. Valid annotation keys have two segments: an optional prefix and name, separated by a slash (/). The name segment is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between. The prefix is optional. If specified, the prefix must be a DNS subdomain: a series of DNS labels separated by dots (.), not longer than 253 characters in total, followed by a slash (/).

Reply:
Oh, right, I forgot the key part of the annotation 🤦‍♂